buildpacks/java/java/build.go

```diff
@@ -49,16 +49,16 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return result, nil
 	}
 
-	functionClass, isFuncDefDefault := cr.Resolve("BP_FUNCTION")
+	functionClass, isOverride := cr.Resolve("BP_FUNCTION")
```

Comment:
This variable was named the opposite of what it should be (thanks @andrew-su for pointing this out)